src/redux/modules/cases.ts

```diff
@@ -14,15 +14,20 @@ export const casesActionCreators = {
     action("cases/TOGGLE_DETAILS", receiptNumber),
   setCaseType: (type: SnoozeState) => action("cases/SET_CASE_TYPE", type),
   setIsLoading: (isLoading: boolean) =>
-    action("cases/SET_IS_LOADING", isLoading)
+    action("cases/SET_IS_LOADING", isLoading),
+  setCaseSummary: (summary: Summary) =>
+    action("cases/SET_CASE_SUMMARY", summary),
+  setLastUpdated: (lastUpdated: string) =>
+    action("cases/SET_LAST_UPDATED", lastUpdated)
 };
 
 export const loadCases = (
   type: SnoozeState,
   lastReceiptNumber?: string
-) => async (dispatch: Dispatch<AnyAction>) => {
+) => async (dispatch: Dispatch<any>) => {
```

Comment:
Using `Dispatch<any>` here because the redux typing isn't quite right for dispatching thunks. Need to look into how to be able to dispatch either an action or a thunk without resorting to `any`.